static/i18n/en/app.json

```diff
@@ -5,6 +5,7 @@
     "apply": "Apply",
     "confirm": "Confirm",
     "cancel": "Cancel",
+    "getSupport": "Get Support",
```

Comment:
we want this third wording @dasilvarosa  ? Contact us / Contact Support / Get Support 🤔